src/WizardLayout/WizardLayout.less

```diff
@@ -0,0 +1,105 @@
+@import (reference) "../less/utilities";
+
+.WizardLayout {
+  .borderRadius--m;
```

Comment:
nit for CSS files: add parentheses when calling mixins. Without parentheses is going to be deprecated http://lesscss.org/features/#mixins-feature